app/src/common/shared/com/igalia/wolvic/browser/engine/Session.java

```diff
@@ -76,7 +76,7 @@ public class Session implements WContentBlocking.Delegate, WSession.NavigationDe
     private static UriOverride sDesktopModeOverrides;
     private static final long KEEP_ALIVE_DURATION_MS = 1000; // 1 second.
 
-    private static final String WOLVIC_USER_AGENT_MOBILE = "Mozilla/5.0 (Android 10; Mobile; rv:105.0) Gecko/105.0 Wolvic/" + BuildConfig.VERSION_NAME;
+    private static final String WOLVIC_USER_AGENT_MOBILE = "Mozilla/5.0 (Android 10; Mobile; rv:105.0) Gecko/105.0 Wolvic/" + BuildConfig.VERSION_NAME + " Firefox/105.0 OculusBrowser/25.3.0.4.30.438623098 VR Safari/537.36";
```

Comment:
Sorry for the back and forth, but it looks like we don't need that.

So we'll stick to the plan of having Gecko/x.x Firefox/x.x by default and then letting users to opt-in to Gecko/x.x Wolvic/x.x in developer options.

Reply:
Change reverted. This is how the User-Agent works now:

- Not Wolvic's own UA, _VR_ mode (default): 
  `Mozilla/5.0 (Android 10; Mobile VR; rv:105.0) Gecko/105.0 Firefox/105.0`
- Not Wolvic's own UA, _Mobile_ mode:
  `Mozilla/5.0 (Android 10; Mobile; rv:105.0) Gecko/105.0 Firefox/105.0`
- Wolvic's own UA, _VR_ mode:
  `Mozilla/5.0 (Android 10; Mobile VR; rv:105.0) Gecko/105.0 Wolvic/1.2`
- Wolvic's own UA, _Mobile_ mode:
  `Mozilla/5.0 (Android 10; Mobile; rv:105.0) Gecko/105.0 Wolvic/1.2`